frontend/package.json

```diff
@@ -6,6 +6,10 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
+    "copy-config": "mkdir -p ./src/generated; cp ../config.json ./src/generated/",
+    "postinstall": "yarn copy-config",
+    "prestart": "yarn copy-config",
+    "prebuild": "yarn copy-config",
```

Comment:
1. This is the first step of the PR, here we are adding the `copy-config` script that will be executed when we start or build the application

Reply:
In the postinstall we add it to avoid importing issues, issues of the type notFound because it does not find the config.json